content/faculty/partners.md

```diff
@@ -0,0 +1,69 @@
+title: Наши партнеры
+slug: partners
+category: faculty
+tags: [_nav]
+---
+
+##"ОМА"##
+
+Компания ОМА была основана в марте 1992 года и на сегодняшний день является лидером по продаже строительных материалов и инструментов, товаров для дома, сада и огорода на рынке Беларуси.
+Компания имеет филиалы во всех областных центрах Беларуси, а также обширную розничную сеть. Штат сотрудников компании составляет более 1500 человек.
+
+<a href="http://www.oma.by/"><img src = "/img/content/partners/oma.png" alt="ОМА"></a>
```

Comment:
It's markdown. Use `[]()` instead. http://daringfireball.net/projects/markdown/basics